main.py

```diff
@@ -4,6 +4,8 @@
 import logging
 import os
 
+import pandas as pd
```

Comment:
Nit: Doesn't seem like `pandas` is used.

Reply:
Oh, thanks! I went back and forth on whether to use a dataframe or not json.